recipes-kernel/linux-firmware/linux-firmware_git.bb

```diff
@@ -138,6 +138,9 @@ S = "${WORKDIR}/git"
 
 inherit allarch update-alternatives
 
+# OE started complaining about the architecture of the binaries for some reason
+INSANE_SKIP = "arch"
```

Comment:
[upstream did too](https://github.com/openembedded/openembedded-core/commit/292eef00b889c61d91c3c965fd3a1009a85f7aa7)

Apparently some recent firmwares started triggering the QA check.